lib/peek/rblineprof/controller_helpers.rb

```diff
@@ -17,13 +13,9 @@ module ControllerHelpers
 
       protected
 
-      def pygmentize?
-        defined?(Pygments)
-      end
-
-      def pygmentize(file_name, code, lexer = nil)
-        if pygmentize? && lexer.present?
-          Pygments.highlight(code, :lexer => lexer_for_filename(file_name))
+      def highlight(file_name, code, lexer = false)
+        if lexer
+          SyntaxHighlighter.highlight(code, lexer_for_filename(file_name))
```

Comment:
I would suggest passing `SyntaxHighlighter.highlight(file_name, code)` and let `SyntaxHighlighter` handle the `lexer_for_filename(file_name)` part since:

- it's not the responsibility of the controller helper
- it will allow to monkey-patch `SyntaxHighlighter` more easily